scripts/init-node.sh

```diff
@@ -96,6 +111,7 @@ function run_node() {
       "--skip-auto-port-forwarding " \
       "--root-dir ~/node_data " \
       "--log-dir ~/logs " \
+      "--features otlp"
```

Comment:
I think the node needs to be built with `--features otlp` rather than launched with it?

Reply:
You're right.. Fixing it!